doc/doc-site/docs/off-chain/dss/gcp.md

```diff
@@ -0,0 +1,916 @@
+       --port 3000 \
+       --set-env-vars "$(printf '%s' \
+           PGRST_DB_ANON_ROLE=web_anon,\
+           PGRST_DB_SCHEMA=api,\
+           PGRST_DB_URI=$DB_URL_PRIVATE\
+       )" \
+       --vpc-connector postgrest
+   ```
+
+1. Store the [service URL](https://cloud.google.com/run/docs/reference/rest/v1/namespaces.services#ServiceStatus) in a shell variable:
+
+   ```sh
+   export REST_URL=$(
+       gcloud run services describe postgrest \
+           --format "value(status.url)"
+   )
+   echo $REST_URL
+   ```
+
+1. Verify that you can query the PostgREST API from the public URL:
+
+   ```sh
+   curl $REST_URL
+   ```
+
+### Deploy processor
+
+1. Create a config at `econia/src/docker/processor/config.yaml` per the [general DSS guidelies](./data-service-stack.md):
+
+   :::tip
+   For `postgres_connection_string` use the same one that the `postgrest` service uses:
+
+   ```sh
+   echo $DB_URL_PRIVATE
+   ```
+
+   :::
+
+1. Upload the processor config to the bootstrapper:
+
+   ```sh
+   gcloud compute scp \
+       econia/src/docker/processor/config.yaml \
+       bootstrapper:~ \
+       --ssh-key-file ssh/gcp
+   ```
+
+1. Connect to the bootstrapper:
+
+   ```sh
+   gcloud compute ssh bootstrapper --ssh-key-file ssh/gcp
+   ```
+
+1. Create a processor data directory and move the config file into it:
+
+   ```sh
+   mkdir /mnt/disks/processor/data
+   mv config.yaml /mnt/disks/processor/data/config.yaml
+   ```
+
+1. End the connection with the bootstrapper:
+
+   ```
+   exit
+   ```
+
+1. Stop the bootstrapper:
+
+   ```sh
+   gcloud compute instances stop bootstrapper
+   ```
+
+1. Detach `processor-disk` from the bootstrapper:
+
+   ```sh
+   gcloud compute instances detach-disk bootstrapper --disk processor-disk
+   ```
+
+1. Deploy the `processor` image:
+
+   ```sh
+   gcloud compute instances create-with-container processor \
+       --container-env HEALTHCHECK_BEFORE_START=false \
+       --container-image \
+           $REGION-docker.pkg.dev/$PROJECT_ID/images/processor:latest \
+       --container-mount-disk "$(printf '%s' \
+           mount-path=/config,\
+           name=processor-disk\
+       )" \
+       --disk "$(printf '%s' \
+           auto-delete=no,\
+           device-name=processor-disk,\
+           name=processor-disk\
+       )"
+   ```
+
+1. Give the processor a minute or so to start up, then [view the container logs](https://cloud.google.com/compute/docs/containers/deploying-containers#viewing_container_logs):
+
+   ```sh
+   PROCESSOR_ID=$(gcloud compute instances describe processor \
+       --zone $ZONE \
+       --format="value(id)"
+   )
+   gcloud logging read "resource.type=gce_instance AND \
+       logName=projects/$PROJECT_ID/logs/cos_containers AND \
+       resource.labels.instance_id=$PROCESSOR_ID" \
+       --limit 5
+   ```
+
+1. Once the processor has had enough time to sync, check some of the events:
+
+   ```sh
+   curl $REST_URL/<AN_ENDPOINT>
```

Comment:
I'd provide some example endpoints and a comment saying they're not exhaustive.

Reply:
Addressed in f65f664